gips/data/cdl/cdl.py

```diff
@@ -67,29 +80,48 @@ class cdlData(Data):
     _products = {
         _cdl: {
             'description': 'Crop Data Layer',
-            'assets': [_cdl],
+            'assets': [_cdl, _cdlmkii],
             'bands': [{'name': _cdl, 'units': 'none'}],
             # presently 'startdate' & 'latency' are permitted to be unspecified
             # by DH gips.utils.get_data_variables
         }
     }
 
-    _legend = None
+    def process(self, products, overwrite=False, **kwargs):
```

Comment:
Should this use the new tempdir code for its first iteration?  If so, I think most drivers have it now if you want something to work from.